products/extensions/b2b-components/employee-management/concept/README.md

```diff
@@ -0,0 +1,16 @@
+# Concept
+
+## Basic idea
+
+Employee management, as part of the B2B components, is a feature that allows you to manage employees and their permissions as an extension to Shopware's account and customer management, but gathered in a company context. This means that employees are associated with a **company customer** and will act on behalf of that company, when talking about e.g. placing orders in the shop using addresses and payment methods defined by administrators of such company.
+For that matter, we needed a way to provide order relevant data without building the checkout process anew and decided on using a **company customer**
+
+## Company customer
+
+The company customer is a regular customer in the shop, but with a few additional properties. This customer's ID is used to associate employees with a company. Therefore, it is really easy to relate on Shopware's typical order relevant data like addresses and payment methods.
+Internally, every employee will act like it is the company customer itself, using mostly the default handling of Shopware's Core.
+The entities of the B2B components will persist additional like the name of the employee in addition for internal purposes like the distinction of employee actions.
```

Comment:
I guess we can drop this line, since we have the entity overview already. Otherwise, please re-read and re-phares it 😄